packages/lib/.stylelintrc.json

```diff
@@ -4,6 +4,7 @@
         "no-descending-specificity": null,
         "selector-class-pattern": null,
         "scss/no-global-function-names": null,
-        "declaration-block-no-redundant-longhand-properties": null
+        "declaration-block-no-redundant-longhand-properties": null,
+        "media-feature-range-notation": null
```

Comment:
What does this new rule do?

Reply:
https://stylelint.io/user-guide/rules/media-feature-range-notation/

As I updated stylelint, it started complaining about some media rules. I disabled the rule as it didn't seem like a big deal. Let me know if you think the opposite